cmd/rqlite/execute.go

```diff
@@ -21,9 +21,9 @@ type executeResponse struct {
 }
 
 func execute(ctx *cli.Context, cmd, line string, argv *argT) error {
-	urlStr := fmt.Sprintf("%s://%s:%d/db/execute?pretty&timings", argv.Protocol, argv.Host, argv.Port)
+	urlStr := fmt.Sprintf("%s://%s:%d%sdb/execute?pretty&timings", argv.Protocol, argv.Host, argv.Port, argv.Prefix)
```

Comment:
What exactly is the point of this change? I don't follow what one would use this switch for.